packages/dara-core/dara/core/visual/template.py

```diff
@@ -44,7 +44,7 @@ def add_route(
         content: ComponentInstance,
         icon: Optional[str] = None,
         include_in_menu: Optional[bool] = True,
-        reset_vars_on_load: Optional[List[AnyVariable]] = [],
+        on_load: Optional[Action] = None,
```

Comment:
Is this technically a breaking change?

Reply:
Both `reset_vars_on_load` and `on_load` exist on the public API `config.add_page`. 

We have to draw the line on what's "internal" vs "public API" somewhere, and I'd consider this template API more internal. In theory since this is Python you can import any of Dara internals and use them. 

In this case, even the custom templates we use e.g. in the CLI do not use this API directly. I might be wrong though and this might be a commonly used API, happy to retain the old kwarg here if that's the case.

Reply:
makes sense, I'm ok with that. Not sure it is often used